### PR TITLE
Updated topology page.

### PR DESCRIPTION
**Pull request by woutslakhorst**
- updated graphics
- mouse over action giving additional info on a node
- info on node: NodeDID, certificate info, NutsComm address, tx count, software info and more

Screenshot 2023-05-02 at 13 29 00

### CODE REVIEW

File: client/topology.go [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+					peer.ContactWeb = nodeContactInfo.Web
+					peer.ContactPhone = nodeContactInfo.Phone
+
+					nutsComm, err := extractNutsCommAddress(document.Document)
+					if err != nil {
+						logrus.Errorf("failed to extract NutsComm address from DID Document: %v", err)
+					} else {
+						wgTLS.Add(1)
+						go func(peer *Peer) {
+							conn, err := tls.Dial("tcp", nutsComm, &tls.Config{InsecureSkipVerify: true})
```
Comment (reinkrul) [inline client/topology.go:189]: what is the time-out?

File: client/topology.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+// addInfoToPeers adds information from DID Documents and the Certificate exposed at the NutsComm address.
+func (ts TopologyService) addInfoToPeers(ctx context.Context, peers []Peer) {
+	wgDocument := sync.WaitGroup{}
+	wgTLS := sync.WaitGroup{}
+	for i, p := range peers {
+		if p.Authenticated {
+			// connect to the node and certificate info
+			wgDocument.Add(1)
+			go func(peer *Peer) {
+				document, err := ts.HTTPClient.DIDDocument(ctx, *peer.NodeDID)
```
Comment (reinkrul) [inline client/topology.go:173]: You might want to cache this, since this is an N+1 type query?
(same for the TLS dial)

**Review by reinkrul — CHANGES_REQUESTED** [review]
I don't have an opinion on the javascript part of things

File: client/topology.go (hunk 1 above)
Comment (woutslakhorst) [reply client/topology.go]: changed it to 2 second.

**Review by reinkrul — APPROVED** [review]

**Pull request merged**